### PR TITLE
refactor: move `getCurrentCurrency` from `ui/selectors/selectors.js` to `ui/ducks/metamask/metamask.js`

### PR DESCRIPTION
**Pull request by davidmurdoch**
This change is related to circular dependency work; no runtime code has changed. QA is not required.

This PR moves `getCurrentCurrency` into `ui/ducks/` instead of keeping it in `ui/selectors/` because `ui/ducks/metamask/metamask.js` already exports a `getNativeCurrency`, so it seems to fit in well there.

### CODE REVIEW

**Comment by Jcisneros618** [comment]
> **CLA Signature Action:** All authors have signed the CLA. You may need to manually re-run the blocking PR check if it doesn't pass in a few minutes.

File: ui/pages/confirmations/components/simulation-details/useBalanceChanges.test.ts [hunk 1]
```diff
@@ -17,9 +17,11 @@ jest.mock('react-redux', () => ({
   useSelector: jest.fn((selector) => selector()),
 }));
 
-jest.mock('../../../../selectors', () => ({
-  getCurrentChainId: jest.fn(),
```
Comment (davidmurdoch) [inline ui/pages/confirmations/components/simulation-details/useBalanceChanges.test.ts:21]: I think `getCurrentChainId` just wasn't being used.

File: ui/selectors/selectors.js [hunk 2]
```diff
@@ -956,10 +956,6 @@ export function getNftIsStillFetchingIndication(state) {
   return state.appState.isNftStillFetchingIndication;
 }
 
-export function getCurrentCurrency(state) {
-  return state.metamask.currentCurrency;
-}
-
```
Comment (davidmurdoch) [inline ui/selectors/selectors.js:962]: all these changes just for this one function to move :-)

**Review by HowardBraham — DISMISSED** [review]

**Review by HowardBraham — DISMISSED** [review]

**Review by HowardBraham — APPROVED** [review]

**Review by Gudahtt — APPROVED** [review]
LGTM!

**Pull request merged**